placement/selector/mirrored.go

```diff
@@ -419,42 +419,42 @@ func (h host) addInstance(port uint32, instance placement.Instance) error {
 		return fmt.Errorf("could not add instance %s to host %s, weight mismatch: %d and %d",
 			instance.ID(), h.name, instance.Weight(), h.weight)
 	}
-	if h.rack != instance.Rack() {
-		return fmt.Errorf("could not add instance %s to host %s, rack mismatch: %s and %s",
-			instance.ID(), h.name, instance.Rack(), h.rack)
+	if h.ig != instance.IsolationGroup() {
+		return fmt.Errorf("could not add instance %s to host %s, isolation group mismatch: %s and %s",
+			instance.ID(), h.name, instance.IsolationGroup(), h.ig)
 	}
 	h.portToInstance[port] = instance
 	return nil
 }
 
-type rack struct {
-	rack  string
+type ig struct {
```

Comment:
Yeah we should probably call these `type isoGroup struct` or something and put a comment above it.